website/docusaurus.config.js

```diff
@@ -41,7 +41,25 @@ module.exports = {
         { to: 'https://github.com/h2oai/wave/discussions', label: 'Discuss', position: 'left' },
         { to: 'https://h2oai.github.io/h2o-ai-cloud/', label: 'Enterprise', position: 'left', },
         { to: 'https://github.com/h2oai/wave/releases', label: 'Download', position: 'right', },
-        { href: 'https://github.com/h2oai/wave', label: 'GitHub', position: 'right', },
+        { href: 'https://github.com/h2oai/wave',
+          position: 'right',
+          className: 'navbar-icon navbar-icon-github',
```

Comment:
Are `navbar-icon-<social-media>` defined somewhere? If not, let's remove those. Seems like only `navbar-icon` class is used.

Same for the rest of the icons.

```suggestion
          className: 'navbar-icon',
```